### DISCUSSION TITLE
Does Xamarin support Xcode version 14 (iOS 16 simulators)?

### DISCUSSION THREAD

**Discussion by mmholtman** [xamarin/Xamarin.Forms#15508] [General]
Visual Studio 2022 doesn't appear to recognize any simulators from the Xcode version 14 beta...I don't see anything in the VS 2022 release notes that say it supports iOS 16. Curious if there is a plan to support it or if I could be missing something?

**Comment by giuseppenovielli** [comment]
@mmholtman 
https://dotnet.microsoft.com/en-us/platform/support/policy/xamarin#:~:text=Xamarin%20support%20will%20end%20on,final%20versions%20Xamarin%20will%20target.

'Xamarin support will end on May 1, 2024 for all Xamarin SDKs. Android 13 and Xcode 14 SDKs (iOS and iPadOS 16, macOS 13) will be the final versions Xamarin will target.'

**Comment by mohachouch** [comment]
The release of Android 14 will be available in September 2023 on Google Pixel. Applications on Xamarin Forms will therefore not be able to work on Pixel device from September 2023. It is therefore necessary either to change the end of support date to September 2023 or to make Xamarin Forms 5.0 compatible with Android 14. @jfversluis  @davidortinau @PureWeen @jsuarezruiz  Thank you.
